app/assets/javascripts/admin/addon/components/admin-area-settings.gjs

```diff
@@ -69,6 +75,7 @@ export default class AdminAreaSettings extends Component {
 
     <div
       class="content-body admin-config-area__settings admin-detail pull-left"
+      {{didUpdate this.reloadSettings @plugin}}
```

Comment:
I wanted to rewire the data flow here to work reactively, but that was a very deep rabbit hole that I think deserves its own PR. It seems this is a simple enough workaround for now. 🙏

Reply:
I think this makes sense for now :+1: